integrations/cohere/src/cohere_haystack/generator.py

```diff
@@ -4,13 +4,11 @@
 import logging
 import os
 import sys
-from typing import Any, Callable, Dict, List, Optional
+from typing import Any, Callable, Dict, List, Optional, cast
 
+from cohere import COHERE_API_URL, Client
+from cohere.responses import Generations
 from haystack import DeserializationError, component, default_from_dict, default_to_dict
-from haystack.lazy_imports import LazyImport
-
-with LazyImport(message="Run 'pip install cohere'") as cohere_import:
```

Comment:
I see you're removing this because cohere is installed as a dependency of the cohere-haystack package anyway. 👍 Let's clean this up in the other integrations then too otherwise people might use them as an example: https://github.com/search?q=repo%3Adeepset-ai%2Fhaystack-core-integrations%20LazyImport&type=code Could you please create an issue?

Reply:
Yup, here you go! https://github.com/deepset-ai/haystack-core-integrations/pull/83